test/unit/stores/test_image.py

```diff
@@ -82,3 +84,54 @@ def test_upload(self):
         self.mock_upload(IMAGE_POST_METADATA_DATA_ALL_VALUES)
         self.call_upload()
         self.check_upload_success()
+
+
+class CreatePresignedURLDSL(ImageStoreDSL):
+    """Base class for `create` tests."""
+
+    _image: ImageOut
+    _expected_presigned_url: HttpUrl
+    _obtained_presigned_url: HttpUrl
+
+    def mock_create_presigned_get(self) -> None:
```

Comment:
```suggestion
    def mock_create_presigned_get(self, image_in_data: dict) -> None:
```
I would handle this like the repo and take in the data that it will test with and use that to construct the image out model so future tests could change it.